x-pack/filebeat/input/azureblobstorage/job.go

```diff
@@ -175,6 +175,7 @@ func (j *job) readJsonAndPublish(ctx context.Context, r io.Reader, id string) er
 			if err := j.splitEventList(j.src.ExpandEventListFromField, item, offset, j.hash, id); err != nil {
 				return err
 			}
+			continue
```

Comment:
Anyway to hold this in a test? Looks like if we remove this it still goes through

Reply:
@bhapas the nature of this test is such that right now it's not possible to do a negative fail test with the current test suit. I will look into how we can improve the test suit to incorporate such negative tests in future.

Reply:
Can we test this manually , if automation suite does not support now?

Reply:
yes manually tested this on my local already and is behaving as expected. This is basically a copy of the same logic present in the s3 input here: https://github.com/elastic/beats/blob/a55ae5857c0d80e0834c9f3e98fd1b833675f12c/x-pack/filebeat/input/awss3/s3_objects.go#L220

Reply:
Great...!